game-app/game-core/src/main/java/games/strategy/engine/delegate/IDelegateBridge.java

```diff
@@ -99,7 +93,11 @@ public interface IDelegateBridge {
 
   void sendMessage(WebSocketMessage webSocketMessage);
```

Comment:
@DanVanAtta This method is unused but seems to have to do with the server relay project. Should it be removed or will it serve a certain purpose in the near future?